app.cr

```diff
@@ -14,10 +14,26 @@ def headers
   headers
 end
 
-def crystal_repos(sort)
+def crystal_repos
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
-  GithubRepos.from_json(response.body)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  total_repos = GithubRepos.from_json(response.body).total_count
+  repos = GithubRepos.from_json(response.body)
```

Comment:
Do we have to parse the response twice? Can this be "cached" in a local variable?